src/components/NcRichText/NcReferencePicker/NcReferencePickerModal.vue

```diff
@@ -186,4 +186,13 @@ export default {
 		}
 	}
 }
+// Hide the empty modal header which prevents from accessing the close button
+// Cannot use display:none because there's a `display: flex !important` in the
+// Ncmodal component which would override it. This works because the
+// `.modal-header` element has `position: absolute`.
+// TODO: Add prop to NcModal to hide the header
```

Comment:
If this is a dialog then it should use `NcDialog` and not `NcModal` which also would fix this issue.